proposals/0310-PerformInteractionMultipick.md

```diff
@@ -1,251 +1,232 @@
-    <description>Triggers an interaction where a user can pick multiple things (e.g. "What do you want to order from this restaurant?"). </description>
-    <param name="initialText" type="Common.TextFieldStruct" mandatory="false">
-      <description>Uses initialInteractionText. See TextFieldStruct.</description>
-    </param>
-    <param name="multiPickConfirmationButtonText" type="String" maxlength="500" mandatory="false">
-      <description>
-        This text will be the button that a person uses to finish choosing items.
-      </description>
-    </param>
-    <param name="choiceSet" type="Common.Choice" minsize="1" maxsize="100" array="true" mandatory="false">
-      <description>The list of choices to be used for the interaction with the user</description>
-    </param>
-    <param name="timeout" type="Integer" minvalue="5000" maxvalue="100000" defvalue="10000" mandatory="true">
-      <description>Timeout in milliseconds.</description>
-    </param>
-    <param name="interactionLayout" type="Common.LayoutMode" mandatory="false">
-      <description>See LayoutMode.</description>
-    </param>
-    <param name="appID" type="Integer" mandatory="true">
-      <description>ID of application that concerns this RPC.</description>
-    </param>
-    <param name="cancelID" type="Integer" mandatory="false">
-      <description>
-        An ID for this specific alert to allow cancellation through the `CancelInteraction` RPC.
-      </description>
-    </param>
-  </function>
-</interface>
-```
-```xml
-  <function name="PerformInteractionMultipick" messagetype="response">
-    <param name="choiceID" type="Integer" minsize="1" maxsize="100" minvalue="0" maxvalue="2000000000" array="true" mandatory="false">
-      <description>IDs of the choices that were selected. Allows duplicate choiceIDs so that the HMI can send back the same choiceID multiple times to relay a quantity back to the app.
-
-          Only is valid if general result is "success:true".</description>
-    </param>
-  </function>
-```
+# Perform Interaction Multipick
+
+* Proposal: [SDL-0310](0310-PerformInteractionMultipick.md)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Returned for Revisions**
+* Impacted Platforms: [HMI / Core / iOS / Java Suite / JavaScript / RPC ]
+
+## Introduction
+
+For when a user wants to order something from an app, the user may want to pick 1 or more of their favorite items from a list. This would allow that. Currently, perform interaction only accepts 1 input and clears.
+
+## Motivation
+
+As a user, I want to be able to pick multiple things from a list of my favorite orders so that I can order everything I want through the head unit.
+
+Here's an example from an overall plan
+
+<img src="../assets/proposals/0310-PerformInteraction-Multipick/PerformInteraction Multipick overview.PNG" alt="Login Screen" class="inline" height= "100%" width= "100%" /> 
+
+## Proposed solution
```

Comment:
The following was described as a potential part of the  `MANUAL_SELECT`/`InteractionMode` solution that these revisions were requested to incorporate (https://github.com/smartdevicelink/sdl_evolution/issues/1052#issuecomment-676497507). Can you please add if this is still required for this proposed solution, or advise why this is no longer needed?
>The enum `BOTH` could have a reworded description to state it is a manual and VR interaction and does not include the multiselect option. We could also give these values new internal name values that better describe their purposes.

Reply:
This isnt needed with the new proposal becasue the `InteractionMode` will be `BOTH` or `MANUAL_MULTISELECT`. It's kind of intuitive that the only way to get a Multiselect PI would be to set the interaction mode to `MANUAL_MULTISELECT`